### PR TITLE
Fix edge cases, follow redirects

### PR DESCRIPTION
**Pull request by aurabirb**

### CODE REVIEW

File: src/extract_bootstrap_from_bundles.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const extractBootstrapFromBundles = async ({
   const gettingBundles = makeProgress('Getting bundles');
   let scriptsContents;
   try {
-    scriptsContents = await Promise.all(bundleUrls.map((url) => httpsGet(url, projectName)));
+    scriptsContents = (await Promise.allSettled(bundleUrls.map((url) => httpsGet(url, projectName)))).map(({value}) => value || "");
```
Comment (wojtekmaj) [inline src/extract_bootstrap_from_bundles.js]: Will httpsGet return { value } when rejecting? Why Promise.allSettled?

Comment (aurabirb) [reply src/extract_bootstrap_from_bundles.js]: Yes, this works even if the page returns a 404. Return value will be something like `[{status: "fulfilled", value: 99}, {status: "rejected",  reason: Error: an error}]` therefore value === undefined. Then this `undefined` turns into an empty string.

**Comment by wojtekmaj** [comment]
Lmao at commit message

Thanks, approved!

**Comment by wojtekmaj** [comment]
Lmao at commit message

Thanks, approved!

**Pull request merged**